dev/NavigationView/NavigationView.xaml

```diff
@@ -399,7 +399,11 @@
                                                     <ScrollViewer VerticalScrollBarVisibility="Auto">
                                                         <local:ItemsRepeater
                                                                     AutomationProperties.AccessibilityView = "Content"
-                                                                    x:Name="TopNavMenuItemsOverflowHost"/>
+                                                                    x:Name="TopNavMenuItemsOverflowHost">
+                                                            <local:ItemsRepeater.Layout>
+                                                                <local:StackLayout />
```

Comment:
Rather than make 3 different stack layouts per NavigationView, could we instead make one stack layout, that is stored in the template root's resources and is referenced in these three locations?

Reply:
Its better to not share these since we are touching some of the properties.